code/__DEFINES/mobs.dm

```diff
@@ -564,7 +564,11 @@ GLOBAL_LIST_INIT(xenoupgradetiers, list(XENO_UPGRADE_BASETYPE, XENO_UPGRADE_INVA
 #define CRUSHER_CHARGE_RAZORWIRE_MULTI	100
 #define CRUSHER_CHARGE_TANK_MULTI		100
 
-#define CRUSHER_STOMP_UPGRADE_BONUS ( 1 + upgrade_as_number() * 0.05 )
+#define CHARGE_TURFS_TO_CHARGE			5		//Amount of turfs to build up before a charge begins
+#define CHARGE_SPEED_BUILDUP			0.15 	//POSITIVE amount of speed built up during a charge each step
```

Comment:
This is duplicated in #1233

Reply:
Yeah will resolve conflicts with it after that's merged.